dockstore-webservice/src/main/resources/migrations.1.6.0.xml

```diff
@@ -235,4 +235,12 @@
                       tableName="event"/>
         <sql dbms="postgres">ALTER SEQUENCE organisation_id_seq RENAME TO organization_id_seq</sql>
     </changeSet>
+
+    <changeSet author="dyuen" id="maintain_consistency">
+        <!-- looks like the above attempt didn't take -->
```

Comment:
Yeah, I don't understand that... I had the same problem for the absolute path where it didn't seem to take with `<sql dbms` either. It's not clear to me why they have that.